test/Microsoft.AspNetCore.Sockets.Tests/ConnectionManagerTests.cs

```diff
@@ -101,6 +101,31 @@ public void RemoveConnection()
             await state.DisposeAsync();
         }
 
+        [Fact]
+        public async Task DisposeAndRemoveAsyncWithInactiveConnection()
```

Comment:
Why don't you just create a connection and call DisposeAsync without setting anything?